ctype.md

```diff
@@ -2,50 +2,48 @@
 
 Before the <span class="label-role claimer">claimer</span> can make a claim about themselves, they first need a claim type (CTYPE for short).
 
-A claim type (CTYPE for short) is a KILT-specific term, but the concept is simple.  
+A claim type (CTYPE for short) is a KILT-specific term, but the concept is simple:  
 A CTYPE defines the structure of a claim. You can think of it as the data model for your claim.
 
 For example, a very basic CTYPE for a driver's license could look like this:
 
 ```json
 {
   "schema": {
-    "$id": "KILT_DRIVERS_LICENSE",
+    "$id": "SimpleLicense",
     "$schema": "http://kilt-protocol.org/draft-01/ctype#",
     "properties": {
```

Comment:
With the inclusion of the nested `ctype`s we will be adding `title`. 

I am not sure if you would like to add it now or make it another ticket for after.

Reply:
Good point, let's do when the nested CTYPEs are merged. 
By the way, that's a good occasion to do the following, as discussed a few weeks ago during our chat about documentation: 
change the checkbox text in the SDK PR template:
"I have documented the changes where needed" --> "I have documented the changes where needed: workshop, code examples in the SDK, ...."

Reply:
I think so too. It would be good to do it or at least created a ticket for it to be done.

Reply:
Isn't it part of a documentation ticket you've created earlier?
Otherwise I agree, let's create one. Let me know.

Reply:
I do not know. 

I think the tickets themselves need a clean up.

Reply:
https://github.com/KILTprotocol/ticket/issues/339
Agreed - I think you're owning the DOC board and associated topics --> feel free to edit or delete `documentation`-labelled tickets (also mine) that you feel should be!